lib/actions/rootActions.js

```diff
@@ -91,9 +94,16 @@ export const loadDashboardInfo = id => dispatch => {
   )
     .then(res =>
       res.json().then(data => {
+        const parsingCheck = parseData => {
+          if(!parseData.settings.items){
```

Comment:
shouldn't be `!parseData.settings.items.length`
and `!parseData.data.items.length` below